src/solver/optim-model-filler/include/antares/solver/optim-model-filler/LinearExpression.h

```diff
@@ -52,22 +55,51 @@ class LinearExpression
     /// Only first expression can have non-zero coefficients, otherwise the result cannot be linear
     LinearExpression operator/(const LinearExpression& other) const;
     /// Multiply linear expression by -1
-    LinearExpression negate() const;
+    LinearExpression operator-() const;
 
     /// Get the offset
-    double offset() const
-    {
-        return offset_;
-    }
+    double offset() const;
 
     /// Get the non-zero coefficients per variable ID
-    std::map<std::string, double> coefPerVar() const
-    {
-        return coef_per_var_;
-    }
+    std::map<std::string, double> coefPerVar() const;
+
+    LinearExpression& operator+=(const LinearExpression& value);
 
 private:
     double offset_ = 0;
     std::map<std::string, double> coef_per_var_;
 };
+
+// time dependent parameter
+class TimeDependentLinearExpression
```

Comment:
why is it in the same .h / .cpp as LinearExpression ?

Reply:
done